node/router/src/heartbeat.rs

```diff
@@ -290,11 +297,11 @@ pub trait Heartbeat<N: Network>: Outbound<N> {
     /// This function attempts to connect to any disconnected trusted peers.
     fn handle_trusted_peers(&self) {
         // Ensure that the trusted nodes are connected.
-        for peer_ip in self.router().trusted_peers() {
+        for peer_addr in self.router().trusted_peers() {
```

Comment:
This should also be `peer_ip` as iirc they are the pre-configured trusted listener addresses the nodes can connect to.